tensorboard/webapp/metrics/views/right_pane/settings_view_component.scss

```diff
@@ -101,9 +101,16 @@ mat-slider {
   align-items: center;
   display: flex;
   cursor: pointer;
+  font-size: inherit;
+  height: 25px;
+  padding-left: 5px;
+  padding-right: 8px;
+  &.toggle-opened {
+    background-color: mat.get-color-from-palette(mat.$grey-palette, 800);
+  }
```

Comment:
Looking at the Settings button, which also has an icon to align, it sets the following on .mat-button-wrapper, which seems to work well:

display: inline-flex;
align-items: center;

Reply:
That works great!